.github/workflows/AddPRComment.yml

```
@@ -1,6 +1,9 @@
 on:
   pull_request_target:
     types: [opened]
+    branches:
+      - main
```

```suggestion
      - dev
```
For azure-cli, we only use dev and release branches.